core/src/main/java/io/lionweb/language/assigners/CommonIDAssigners.java

```diff
@@ -0,0 +1,51 @@
+package io.lionweb.language.assigners;
+
+import io.lionweb.language.INamed;
+import io.lionweb.language.Language;
+import io.lionweb.model.ClassifierInstance;
+import io.lionweb.model.ClassifierInstanceUtils;
+import io.lionweb.model.HasSettableID;
+import io.lionweb.model.Node;
+import io.lionweb.utils.IdUtils;
+import javax.annotation.Nonnull;
+
+public class CommonIDAssigners {
```

Comment:
maybe some javadoc? As a user, I'd like to have some idea what this class considers a "common id".